src/Octoliner.h

```diff
@@ -35,35 +46,41 @@ enum IOcommand {
 
 class Octoliner {
 public:
-    Octoliner();
-    Octoliner(uint8_t i2caddress);
-    void begin(uint8_t value);
-    void digitalWrite(int pin, bool value);
-    void pinMode(int pin, uint8_t mode);
-    void analogWrite(int pin, uint8_t pulseWidth);
-    void pwmFreq(uint16_t freq);
+    Octoliner(uint8_t i2caddress = 42);
+    void begin(uint8_t value = 208);
     void changeAddr(uint8_t newAddr);
     void changeAddrWithUID(uint8_t newAddr);
     void saveAddr();
     void reset();
-    int digitalRead(int pin);
-    int analogRead(int pin); //, uint8_t avgCount = 2);
+    int16_t analogRead(uint8_t pin); //, uint8_t avgCount = 2);
```

Comment:
`//, uint8_t avgCount = 2);` - legacy comment?

Reply:
Current task is made workable version. Comments and legacy code  is future task.

Reply:
Ok